File: app/src/main/java/com/kirawii/thunderswufe/network/ElectricityResponse.java
package com.kirawii.thunderswufe.network;

import com.google.gson.annotations.SerializedName;

public class ElectricityResponse {
    @SerializedName("balance")
    private String balance;

    @SerializedName("status")
    private String status;

    @SerializedName("message")
    private String message;

    public String getBalance() {
        return balance;
    }

    public void setBalance(String balance) {
        this.balance = balance;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        return "ElectricityResponse{" +
                "balance='" + balance + '\'' +
                ", status='" + status + '\'' +
                ", message='" + message + '\'' +
                '}';
    }
} 

File: app/src/main/java/com/kirawii/thunderswufe/network/ElectricityResponse.kt
package com.kirawii.thunderswufe.network

import com.google.gson.annotations.SerializedName // 如果字段名与JSON key不同

/**
 * Data class representing the expected JSON response structure from the API.
 * Fields are nullable to handle potential missing fields in the response.
 */
data class ElectricityResponse(
    // Assuming JSON keys match these field names.
    // If not, use @SerializedName("json_key_name")

    @SerializedName("balance") // Example if JSON key is 'balance'
    val balance: String?,

    @SerializedName("status") // Example
    val status: String?,

    @SerializedName("message") // Example
    val message: String?
    // TODO: Add/remove/modify fields based on the actual API response structure
)